source/includes/fact-nsExclude-precedence.rst

```diff
@@ -0,0 +1,5 @@
+If you specify both ``--nsExclude`` and ``--nsInclude``, the pattern 
+that ``--nsExclude`` specifies takes precedence. For example, if you
+specify both ``--nsExclude="prod.*"`` and ``--nsInclude="prod.trips"``, 
+the pattern specified by ``--nsExclude`` takes precedence and no 
+collections from the ``prod`` namespace are restored. 
```

Comment:
(optional minimalism suggestion) 
To avoid repeating "the pattern specified by ..."
```suggestion
specify both ``--nsExclude="prod.*"`` and ``--nsInclude="prod.trips"``,  no 
collections from the ``prod`` namespace are restored. 
```